### PR TITLE
Makes the faction variable into a list

### PR DESCRIPTION
**Pull request by Tomeno**
This change allows for multiple factions, making mapper lifes easier.
wb(wild-billy) made the CanAttack() proc.

### CODE REVIEW

File: NTstation13.int [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
...hostile\carp.dm;code\modules\mob\living\simple_animal\hostile\creature.dm;code\modules\mob\living\simple_animal\hostile\faithless.dm;code\modules\mob\living\simple_animal\hostile\giant_spider.dm;code\modules\mob\living\simple_animal\hostile\hostile.dm;code\modules\mob\living\simple_animal\hostile\retaliate\bat.dm;code\modules\projectiles\projectile\magic.dm;code\modules\reagents\Chemistry-Recipes.dm;code\modules\events\brand_intelligence.dm;code\modules\mining\equipment_locker.dm;code\modules\mob\living\simple_animal\hostile\hivebot.dm;code\modules\mob\living\simple_animal\hostile\mimic.dm;code\modules\mob\living\simple_animal\hostile\mining_mobs.dm;code\modules\mob\living\simple_animal\hostile\mushroom.dm;code\modules\mob\living\simple_animal\hostile\pirate.dm;code\modules\mob\living\simple_animal\hostile\russian.dm;code\modules\mob\living\simple_animal\hostile\statue.dm;code\modules\mob\living\simple_animal\hostile\syndicate.dm;code\modules\mob\living\simple_animal\hostile\tree.dm;code\modules\mob\living\simple_animal\hostile\void_dweller.dm;code\modules\mob\living\simple_animal\hostile\retaliate\retaliate.dm;code\modules\reagents\reagent_containers\food\snacks.dm;NTstation13.dme
```
Comment (RemieRichards) [inline NTstation13.int]: Revert the changes to this file.
This file should not be changed.

**Comment by WJohn** [comment]
:+1:
This will be a lot of fun for mappers to play with! Thank you <3
